### DISCUSSION TITLE
series-intro-to-github-flow

### DISCUSSION THREAD

**Discussion by swish933** [community/community#114730] [GitHub Education]
### Hi everyone,

I completed the intro to GitHub flow course and it wouldn't mark complete. I left the repo thinking that would make it restart but that wasn't the case and now I have repo access issues

**Comment by Manjunathhs-0003** [comment]
If you completed the introductory GitHub Flow course and it didn't mark as complete, and now you're experiencing repository access issues, here's what you can try:

Check Course Completion Status: Sometimes, completion status may not update immediately. Return to the course page and check if your progress has been marked as complete. If not, try refreshing the page or accessing it from a different browser.

Contact Support: If you're still experiencing issues with course completion or repository access, consider reaching out to GitHub support for assistance. They can help troubleshoot any technical issues you're facing and ensure that your course progress is accurately recorded.

Reset Course Progress: If you're unable to mark the course as complete and it's causing repository access issues, you might consider resetting your course progress. This should reset your progress within the course and allow you to retake it from the beginning.

Repository Access Issues: If you're experiencing access issues to a specific repository, make sure you have the necessary permissions to access it. If you're still having trouble, reach out to the repository owner or GitHub support for assistance.

Clear Cache and Cookies: Sometimes, browser cache and cookies can cause unexpected issues. Try clearing your browser cache and cookies, then attempt to access the course and repository again.

Check Organization Membership: If the repository is part of an organization, make sure you're still a member of that organization. Sometimes, membership changes can affect repository access.

Review GitHub Documentation: GitHub has extensive documentation on common issues and troubleshooting steps. Reviewing the documentation may provide additional insights or solutions to the problems you're encountering.

**Comment by swish933** [comment]
How and where do i reset the course progress?
